packages/hash/README.md

```diff
@@ -6,6 +6,8 @@
  <a href="https://hash.ai/discord?utm_medium=organic&utm_source=github_readme_hash-repo_hash"><img src="https://img.shields.io/discord/840573247803097118" alt="Join HASH on Discord" /></a>
 </div>
 
+<!-- @todo PUT HOW TO REBUILD DOCKER IMAGE -->
```

Comment:
@thehabbos007 had to tell me on a call how to actually test the frontend. I find it quite confusing at the moment that we run Postgres differently depending on if we're just running the graph or not.

I experienced some really strange behaviour where I seemed to manage to run the graph in `external-services` _and_ the graph in `deployment` at the same time without either erroring about ports. This caused problems with the tests because it was redirected to the old image without me knowing 🙃 

Perhaps it's related to this 
<img width="642" alt="image" src="https://user-images.githubusercontent.com/25749103/187701117-04113be6-e983-4074-b339-d4ea94eefcc3.png">

Reply:
@TimDiekmann any thoughts?

Reply:
> I experienced some really strange behaviour where I seemed to manage to run the graph in external-services and the graph in deployment at the same time without either erroring about ports.

I also had this issue and it took me about an hour to figure it out. I don't know how to force docker to error here, I didn't came up with a proper solution.

Generally, to rebuild the graph Docker image you can simply run
```sh
yarn external-services build
```

Reply:
> I find it quite confusing at the moment that we run Postgres differently depending on if we're just running the graph or not.

It is a little confusing, yes. The alternative is to pick external-services as the way we start the postgres container by selecting only the `postgres` service in `cargo make deployment-up`. The reason why we even have multiple setups is because kratos gets its own database, which is created in a postgres init script. 

Having a sort of self-contained dev environment for hash_graph is a little redundant and means extra maintenance, though it does make working on it in isolation a lot simpler.

If